### PR TITLE
Add CRL remediation help

### PR DESCRIPTION
**Pull request by erskinejohn**
We will start blocking CRL issues with validation. 
This needs to be added at the top.

### CODE REVIEW

**Comment by PRMerger16** [comment]
@erskinejohn : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by BryanLa** [comment]
Thanks @erskinejohn .
#sign-off

File: azure-stack/operator/azure-stack-remediate-certs.md [hunk 1]
```diff
@@ -19,6 +19,18 @@ ms.lastreviewed: 10/19/2020
 
 The information in this article helps you understand and resolve common issues with Azure Stack Hub PKI certificates. You can discover issues when you use the Azure Stack Hub Readiness Checker tool to [validate Azure Stack Hub PKI certificates](azure-stack-validate-pki-certs.md). The tool checks if the certificates meet the PKI requirements of an Azure Stack Hub deployment and Azure Stack Hub secret rotation, and then logs the results to a [report.json file](azure-stack-validation-report.md).  
 
+## HTTP CRL - Warning
+
+**Issue** - Certificate does not contain HTTP CRL in CDP Extension.
+
+**Fix** - This is a non-blocking issue. Azure Stack requires HTTP CRL for revocation checking as per https://aka.ms/azspki.  A HTTP CRL was not detected on the certificate.  To ensure certificate revocation checking works, the Certificate Authority should issue a certificate with a HTTP CRL in the CDP extension.
```
Comment (Jak-MS) [inline azure-stack/operator/azure-stack-remediate-certs.md:26]:
```suggestion
**Fix** - This is a non-blocking issue. Azure Stack requires HTTP CRL for revocation checking as per [Azure Stack Hub public key infrastructure (PKI) certificate requirements](https://aka.ms/azspki).  A HTTP CRL was not detected on the certificate.  To ensure certificate revocation checking works, the Certificate Authority should issue a certificate with a HTTP CRL in the CDP extension.
```

File: azure-stack/operator/azure-stack-remediate-certs.md [hunk 2]
```diff
@@ -19,6 +19,18 @@ ms.lastreviewed: 10/19/2020
...
+
+## HTTP CRL - Fail
+
+**Issue** - Cannot connect to HTTP CRL in CDP Extension.
+
+**Fix** - This is a blocking issue. Azure Stack requires connectivity to a HTTP CRL for revocation checking as per Publishing Azure Stack Hub Ports and URLs (outbound) https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-integrate-endpoints#ports-and-urls-outbound.
```
Comment (Jak-MS) [inline azure-stack/operator/azure-stack-remediate-certs.md:32]:
```suggestion
**Fix** - This is a blocking issue. Azure Stack requires connectivity to a HTTP CRL for revocation checking as per [Publishing Azure Stack Hub Ports and URLs (outbound)](https://docs.microsoft.com/azure-stack/operator/azure-stack-integrate-endpoints#ports-and-urls-outbound).
```

**Comment by PRMerger10** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**